lib/apiservers/engine/backends/container.go

```diff
@@ -790,37 +790,41 @@ func (c *Container) containerStop(name string, seconds int, unbound bool) error
 	}
 
 	if unbound {
+		// assume that we have a network configured
+		networkFound := true
 		// get the endpoints for the container
 		conEpsRes, err := client.Scopes.GetContainerEndpoints(scopes.NewGetContainerEndpointsParams().WithHandleOrID(handle))
 		if err != nil {
 			switch err := err.(type) {
 			case *scopes.GetContainerEndpointsNotFound:
```

Comment:
OK - and then if there are no endpoints don't call the other functions -- correct?